src/databricks/labs/ucx/progress/workflows.py

```diff
@@ -22,26 +22,34 @@ class MigrationProgress(Workflow):
     def __init__(self) -> None:
         super().__init__('migration-progress-experimental')
 
-    @job_task
+    @job_task(job_cluster="table_migration")
```

Comment:
why do we need to use `table_migration` cluster instead of the default `main`?

Reply:
I'm not sure about this myself: it was added in #2851.

I did notice that the default cluster is not UC-enabled, which means we can't write to the history file from there, whereas the table-migration cluster is UC-enabled.